README.md

```diff
@@ -262,13 +268,12 @@ make clean ; # cleans up from any previous tests hopefully
 make test-style ; # runs the tests
 make clean ; # cleans up for next test
 ```
-
+---
 ## Next steps:
 
---(UNSTABLE) clean up Proof-of-concept code into a recognizable python module project.--
-(WIP) might expand the documentation to be more user friendly to the non-network guru
-(WIP) might add tcp multicast ... who knows?
+Next-steps and bug-fixes are tracked [Here](https://github.com/users/reactive-firewall/projects/1).
```

Comment:
This improves accuracy by directing interested users to the live version of the information.